src/Cli/dotnet/ProjectInstanceExtensions.cs

```diff
@@ -22,7 +21,7 @@ public static string GetDefaultProjectTypeGuid(this ProjectInstance projectInsta
             string projectTypeGuid = projectInstance.GetPropertyValue("DefaultProjectTypeGuid");
             if (string.IsNullOrEmpty(projectTypeGuid) && projectInstance.FullPath.EndsWith(".shproj", StringComparison.OrdinalIgnoreCase))
             {
-                projectTypeGuid = ProjectTypeGuids.SharedProjectGuid;
+                projectTypeGuid = "{D954291E-2A0B-460D-934E-DC6B0785DB48}";
```

Comment:
Any chance we can make this a constant somewhere?

Reply:
Yes, I want to create a followup PR to centralize project guids, but thought it might be outside the scope of this one